UI/AdvancedXBind/AdvancedXBind/AdvancedXBind/global.json

```diff
@@ -0,0 +1,9 @@
+{
+  // To update the version of Uno please update the version of the Uno.Sdk here. See https://aka.platform.uno/upgrade-uno-packages for more information.
+  "msbuild-sdks": {
+    "Uno.Sdk": "5.3.96"
```

Comment:
This PR will need to be rebased with latest and updated to latest 5.4 stable version
Related new issue: https://github.com/unoplatform/Uno.Samples/issues/828

Reply:
This PR will need to be rebased with latest and updated to latest 5.5 stable version
Related new issue: https://github.com/unoplatform/Uno.Samples/issues/848